### PR TITLE
Pin depepndecy versions

### PR DESCRIPTION
**Pull request by atanaskanchev**

### CODE REVIEW

**Comment by efstathiosntonas** [comment]
@atanaskanchev thanks for the PR, any reason why this patch was downgraded: `"@emotion/react": "11.11.0",`? It's just for version algignment with `emotion/native`?

File: package.json [hunk 1]
```diff
@@ -10,46 +10,46 @@
...
-    "babel-plugin-transform-inline-environment-variables": "^0.4.4",
-    "dripsy": "^4.3.3",
-    "expo": "~49.0.5",
-    "expo-font": "^11.4.0",
-    "expo-splash-screen": "~0.20.4",
-    "expo-status-bar": "~1.6.0",
-    "gluestack-ui": "^0.4.8",
-    "nativewind": "^2.0.11",
+    "@emotion/native": "11.11.0",
+    "@emotion/react": "11.11.0",
```
Comment (efstathiosntonas) [inline package.json:14]: @atanaskanchev thanks for the PR, any reason why this patch was downgraded: "@emotion/react": "11.11.0",? It's just for version alignment with emotion/native?

Comment (atanaskanchev) [reply package.json]: Hi @efstathiosntonas my bad, pushed @emotion/react": "11.11.1

**Comment by efstathiosntonas** [comment]
Thanks @atanaskanchev!

**Pull request merged**